packages/api/src/configuration/index.ts

```diff
@@ -45,24 +45,24 @@ export enum MAX {
   PROPERTY = 24,
   EXPIRESIN = 4,
   ENV = 12,
-  REF = 100,
+  REF = 500,
   CLUSTER = 12
 }
 
 export enum MESSAGE {
-  INVALID_LENGTH = 'Length is Invalid.',
-  INVALID_PROPERTY_TITLE = 'Invalid Property Title.',
-  INVALID_PROPERTY_IDENTIFIER = 'Invalid Property Identifier.',
+  INVALID_LENGTH = 'Length is Invalid',
+  INVALID_PROPERTY_TITLE = 'Invalid Property Title [Example : Test Property].',
+  INVALID_PROPERTY_IDENTIFIER = 'Invalid Property Identifier [Example : test-property].',
```

Comment:
@soumyadip007 why are we mentioning test property?

Reply:
As an example we are writing this.